src/main/java/com/calc/demo/CalculationController.java

```diff
@@ -0,0 +1,93 @@
+package com.calc.demo;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Stack;
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class CalculationController {
+    private static final String template = "Calculated string  , %s!";
+    private final AtomicLong counter = new AtomicLong();
+    private String[] list;
+
+
+    @GetMapping("/a")
+    public Calculation add(@RequestParam(value = "operands", defaultValue = "world") String[] val) {
+
+//        System.out.println(name);
+        for (String c : val) System.out.println(c);
+        double ans = 0;
+        for (String s : val) {
+            ans += (Double.parseDouble(s));
+        }
+
+        return new Calculation(String.format(template, ans));
+
+    }
+
+
+    @GetMapping("/s")
+    public Calculation subtract(@RequestParam(value = "operands", defaultValue = "world") String[] val) {
```

Comment:
why we are taking these as request params ? how defaultvalue "world" will work .. will it not break the code ?